src/Winium.Desktop.Driver/CommandExecutors/SetOrientationExecutor.cs

```diff
@@ -0,0 +1,50 @@
+namespace Winium.Desktop.Driver.CommandExecutors
+{
+    #region using
+
+    using System;
+
+    using Winium.StoreApps.Common;
+    using Winium.Cruciatus.Core;
+
+    #endregion using
+
+    internal class SetOrientationExecutor : CommandExecutorBase
+    {
+        #region Methods
+
+        protected override string DoImpl()
+        {
+            if (!this.ExecutedCommand.Parameters.ContainsKey("orientation"))
+            {
+                // TODO: in the future '400 : invalid argument' will be used
+                return this.JsonResponse(ResponseStatus.UnknownError, "WRONG PARAMETERS");
+            }
+
+            var orientation = (DisplayOrientation)Enum.Parse(
+                typeof(DisplayOrientation), 
+                this.ExecutedCommand.Parameters["orientation"].ToString());
+
+            var result = RotationManager.SetOrientation(orientation);
+
+            switch (result)
+            {
+                case 0:
+                    return this.JsonResponse();
```

Comment:
I am not sure what you are talking about. [JSWP](https://code.google.com/p/selenium/wiki/JsonWireProtocol#/session/:sessionId/orientation) does not state that `POST /session/:sessionId/orientation` should return orientation.